index.html

```diff
@@ -10,6 +10,12 @@
     a.c2 {font-weight: bold}
     span.c1 {color: red;}
     </style>
+    <style type="text/css">
+    a.c1 {font-weight: bold}
+    </style>
+    <style type="text/css">
+    a.c1 {font-weight: bold}
```

Comment:
On further inspection I think that's Tidy trying to remove inline formatting. Good job little bot :) I'll move that over to `style.css`, one sec